lib/versioning/semver/index.js

```diff
@@ -27,16 +28,9 @@ const isRange = input => isValid(input) && !isPinnedVersion(input);
 // If this is left as an alias, inputs like "17.04.0" throw errors
 const isValid = input => validRange(input);
 
-const isLessThan = (version, base) =>
-  isPinnedVersion(base) ? semver.lt(version, base) : semver.ltr(version, base);
```

Comment:
Have you definitely found all the cases where `isLessThan` could be called with a range as base?